src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java

```diff
@@ -2,6 +2,9 @@
 
 import jimmy.mcgymmy.model.Model;
 
+/**
+ * Undo the last command that changed McGymmy.
```

Comment:
I think it should be undoes instead of undo.

I think that changed the ModelManager instead of McGymmy is better